spring-data-sample/docs/README.md

```diff
@@ -0,0 +1,269 @@
+# Spring Data Integration Sample
+
+This tutorial describes how to create a sample Spring Boot application by using Spring Data integration in ScalarDB.
+
+## Prerequisites
+
+- Java (OpenJDK 8 or higher)
+- Gradle
+- Docker, Docker Compose
+
+## Sample application
+
+### Overview
+
+This tutorial describes how to create a sample Spring Boot application for the same use case as [ScalarDB Sample](https://github.com/scalar-labs/scalardb-samples/tree/main/scalardb-sample) but by using Spring Data integration in ScalarDB.
+Please note that application-specific error handling, authentication processing, etc. are omitted in the sample application since this tutorial focuses on explaining how to use Spring Data integration.
+For details, please see [Guide of Spring Data integration with ScalarDB](https://scalar-labs.github.io/scalardb-sql/spring-data-guide.html).
+
+### Schema
+
+[The schema](schema.json) is as follows:
+
+```json
+{
+  "sample.customers": {
+    "transaction": true,
+    "partition-key": [
+      "customer_id"
+    ],
+    "columns": {
+      "customer_id": "INT",
+      "name": "TEXT",
+      "credit_limit": "INT",
+      "credit_total": "INT"
+    }
+  },
+  "sample.orders": {
+    "transaction": true,
+    "partition-key": [
+      "customer_id"
+    ],
+    "clustering-key": [
+      "timestamp"
+    ],
+    "secondary-index": [
+      "order_id"
+    ],
+    "columns": {
+      "order_id": "TEXT",
+      "customer_id": "INT",
+      "timestamp": "BIGINT"
+    }
+  },
+  "sample.statements": {
+    "transaction": true,
+    "partition-key": [
+      "order_id"
+    ],
+    "clustering-key": [
+      "item_id"
+    ],
+    "columns": {
+      "order_id": "TEXT",
+      "item_id": "INT",
+      "count": "INT"
+    }
+  },
+  "sample.items": {
+    "transaction": true,
+    "partition-key": [
+      "item_id"
+    ],
+    "columns": {
+      "item_id": "INT",
+      "name": "TEXT",
+      "price": "INT"
+    }
+  }
+}
+```
+
+All the tables are created in the `sample` namespace.
+
+- `sample.customers`: a table that manages customers' information
+  - `credit_limit`: the maximum amount of money a lender will allow each customer to spend when using a credit card
+  - `credit_total`: the amount of money that each customer has already spent by using the credit card
+- `sample.orders`: a table that manages order information
+- `sample.statements`: a table that manages order statement information
+- `sample.items`: a table that manages information of items to be ordered
+
+The Entity Relationship Diagram for the schema is as follows:
+
+![ERD](images/ERD.png)
```

Comment:
Since this doc contains an image, can we add the "spring-data-sample/images" folder to the "spring-data-sample/docs" folder as well?